mesa/model.py

```diff
@@ -49,20 +49,6 @@ class Model:
         initialize_data_collector: Sets up the data collector for the model, requiring an initialized scheduler and agents.
     """
 
-    def __new__(cls, *args: Any, **kwargs: Any) -> Any:
-        """Create a new model object and instantiate its RNG automatically."""
-        obj = object.__new__(cls)
-        obj._seed = kwargs.get("seed")
-        if obj._seed is None:
-            # We explicitly specify the seed here so that we know its value in
-            # advance.
-            obj._seed = random.random()
-        obj.random = random.Random(obj._seed)
-        # TODO: Remove these 2 lines just before Mesa 3.0
-        obj._steps = 0
-        obj._time = 0
-        return obj
-
     def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
why wrap everything into kwargs, instead of having `seed=None` explicit?

Reply:
That makes sense, I'm changing it to that regardless of whether it fixes the visualization situation or not.